pkg/rehearse/jobs.go

```diff
@@ -482,10 +571,28 @@ func (e *Executor) submitRehearsals() ([]*pjapi.ProwJob, error) {
 		e.loggers.Job.WithFields(pjutil.ProwJobFields(created)).Info("Submitted rehearsal prowjob")
 		pjs = append(pjs, created)
 	}
+
+	for _, job := range e.periodics {
+
+		if !e.allowPeriodics {
+			e.loggers.Job.Warn("Periodics are not allowed to be executed.")
+			break
+		}
```

Comment:
(Assuming `allowPeriodics` is necessary, see above) this can be tested once.

Reply:
it is being testing only once.. I guess its better to put it out from executor and don't include periodics in the `JobConfigurer`